source/extensions/stat_sinks/graphite_statsd/BUILD

```diff
@@ -0,0 +1,29 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+# Stats sink for the Graphite protocol.
+
+envoy_extension_package()
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    category = "envoy.stats_sinks",
+    security_posture = "data_plane_agnostic",
+    status = "alpha",
```

Comment:
hi @vaccarium - you will need to add an entry in `source/extensions/extensions_metadata.yaml` for this extension and re/move `category`, `security_posture` and `status` to there

Reply:
Thank you!